tests/kernel/kdump_over_nfs.pm

```diff
@@ -0,0 +1,70 @@
+# SUSE's openQA tests
+#
+# Copyright 2025 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Configure and run kdump over NFS
+# Maintainer: QE Kernel <kernel-qa@suse.de>
+
+use base "opensusebasetest";
+use strict;
+use warnings;
+use testapi;
+use utils;
+use kdump_utils;
+use lockapi;
+
+sub run {
+    my ($self) = @_;
+    my $role = get_required_var('ROLE');
+    my $kdump_export = get_required_var('KDUMP_SAVEDIR');
+
+    select_console('root-console');
+
+    if ($role eq 'nfs_server') {
+        record_info("KDUMP SHARE: ", script_output("exportfs -v"));
+    }
+
+    if ($role eq 'nfs_client') {
+        prepare_for_kdump();
+        record_info('CONFIG', script_output("cat /etc/sysconfig/kdump"));
+        set_kdump_config('KDUMP_SAVEDIR', $kdump_export);
+        activate_kdump_cli();
+    }
+
+    barrier_wait("KDUMP_PROVISIONED");
```

Comment:
Why do we need `KDUMP` barriers? I think we just need one fixed point only and it is configured NFS server. We don't need any other barriers:
- start nfs server - create barrier or mutex 
- start nfs client - just hang it in grub menu
- nfs server do all configuration and start server
- nfs client - just confirm boot in grub and start
-  do kdump